src/dmd/parse.d

```diff
@@ -6053,10 +6069,14 @@ LagainStc:
             nextToken();
             if (token.value == TOK.assign)
             {
-                error("debug conditions can only be declared at module scope");
-                nextToken();
-                nextToken();
-                goto Lerror;
+                if (auto ds = parseDebugSpecification())
+                {
+                    if (ds.ident)
+                        ds.error("declaration must be at module level");
+                    else
+                        ds.error("level declaration must be at module level");
```

Comment:
Can you add `debug = 1` and `version = 1` to the test case so these else-branches are covered as well?

Reply:
I've added a commit myself